Source/CesiumRuntime/Private/CesiumMetadataPickingBlueprintLibrary.cpp

```diff
@@ -148,23 +150,86 @@ bool UCesiumMetadataPickingBlueprintLibrary::FindUVFromHit(
   return true;
 }
 
+namespace {
+/*
+ * Returns std:nullopt if the component isnt an instanced static mesh or doesn't
+ * have instance feature ID data, which will allow
+ * GetPropertyTableValuesFromHit() to search for feature IDs in primitive
+ * attributes.
+ */
+std::optional<TMap<FString, FCesiumMetadataValue>>
+getInstancePropertyTableValues(
+    const FHitResult& Hit,
+    const UCesiumGltfComponent* pModel,
+    int64 FeatureIDSetIndex) {
+  const auto* pInstancedComponent =
+      Cast<UCesiumGltfInstancedComponent>(Hit.Component);
+  if (!IsValid(pInstancedComponent)) {
+    return std::nullopt;
+  }
+  const TSharedPtr<FCesiumPrimitiveFeatures> pInstanceFeatures =
```

Comment:
Doesn't matter too much, but I think we're doing this in other parts of the code, so do it here for consistency.

```suggestion
  const TSharedPtr<FCesiumPrimitiveFeatures>& pInstanceFeatures =
```